.gitignore

```diff
@@ -102,3 +102,9 @@ dist
 
```

Comment:
I think the .vscode and the .venv code need to be added to the root .gitignore. I had some issue with that too

Reply:
I have included these files in both places: inside and outside